readthedocs/settings/base.py

```diff
@@ -59,7 +59,7 @@ class CommunityBaseSettings(Settings):
     # Email
     DEFAULT_FROM_EMAIL = 'no-reply@readthedocs.org'
     SERVER_EMAIL = DEFAULT_FROM_EMAIL
-    SUPPORT_EMAIL = None
+    SUPPORT_EMAIL = 'support@readthedocs.org'
```

Comment:
Probably we should set it to `support@example.com` and override it on our production settings. That way people using custom installations don't contact us...

Reply:
Agreed. I lived with error emails from private RTD installs for years..

Reply:
PRODUCTION_DOMAIN is `readthedocs.org` so they will still email us :)

I went ahead and use `None`. I also added the PR to change this in -ops for the real email.

Reply:
We should change the default PRODUCTION_DOMAIN as well, then.

Reply:
OK! Talked on Slack. Let's go with `None` for now.